.github/workflows/smoke-checks.yml

```diff
@@ -88,6 +88,49 @@ jobs:
         MATRIX_CONTEXT: ${{ toJson(matrix) }}
       if: ${{ github.event_name == 'push' && failure() }}
 
+  build-and-test-xcode12-debug:
+    name: Test LLC (Xcode 12, Debug)
+    runs-on: macos-11
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set build image var
+      run: echo "ImageVersion=$ImageVersion"  >> $GITHUB_ENV
+    - name: Cache RubyGems
+      uses: actions/cache@v2
+      id: rubygem-cache
+      with:
+        path: vendor/bundle
+        key: ${{ runner.os }}-${{ env.ImageVersion }}-gem-${{ hashFiles('**/Gemfile.lock') }}
+        restore-keys: ${{ runner.os }}-${{ env.ImageVersion }}-gem-
+    - name: Cache Mint
+      uses: actions/cache@v2
+      id: mint-cache
+      with:
+        path: /usr/local/lib/mint
+        key: ${{ runner.os }}-mint-${{ hashFiles('./Mintfile') }}
+        restore-keys: ${{ runner.os }}-mint-
+    - uses: ./.github/actions/bootstrap
+    - name: Run LLC Tests (Debug)
+      env:
+        DEVELOPER_DIR: "/Applications/Xcode_12.5.1.app/Contents/Developer"
+      run: bundle exec fastlane test device:"iPhone 12"
```

Comment:
Can we simply just build the project for now? I don't think it is really worth it, at least for now, to run all the tests on Xcode 12.

Reply:
This will also increase the complexity and will require changes on Codecov. So think, for now, building Xcode 12 is enough to catch these errors.